### PR TITLE
Support post, reply, edit, load and view more loading states via global css classnames

### PR DESCRIPTION
**Pull request by alinhle**
## What does this PR do?
- Changed `coral-load-more-*` css classes to `talk-load-more-*`
- Changed `coral-plugin-commentbox-*` css classes to `talk-plugin-commentbox-*`
- Added classes: `talk-load-more-button-loading`, `talk-load-more-button-error`, `talk-load-more-button-success`.
- Added classes: `talk-plugin-commentbox-button-loading`, `talk-plugin-commentbox-button-success`, `talk-plugin-commentbox-button-error`
- Disable Post, Edit, Reply, Load More, and View More buttons during 'loading' state.
- Only erase comment body when action (post, edit, reply) was successful
- Show error notification when an action was unsuccessful (post, edit, reply, load and view more)
- Animations like this is now possible:
recording 2

## Example animation
```css
@keyframes loadingIndicator {
  0% {
    transform: scale(0);
  } 100% {
    transform: scale(1.0);
    opacity: 0;
  }
}
@keyframes fadeInOut {
  0% {
    opacity: 0;
  }
  50% {
    opacity: 1.0;
  }
  100% {
    opacity: 0;
  }
}
.talk-plugin-commentbox-button-success::before {
  content: '✓';
  display: block;
  position: absolute;
  top: 0;
  left: 0;
  width: 100%;
  height: 100%;
  padding: 3px 0px;
  animation: fadeInOut 2s forwards;
  background: rgb(51, 204, 51);
  color: white;
}
.talk-plugin-commentbox-button-loading::before {
  content: '';
  display: inline-block;
  width: 40px;
  height: 40px;
  position: absolute;
  top: 0px;
  background-color: #333;
  border-radius: 100%;
  animation: loadingIndicator 1.0s infinite ease-in-out;
}
.talk-plugin-commentbox-button-error::before {
  content: '×';
  display: block;
  position: absolute;
  top: 0;
  left: 0;
  width: 100%;
  height: 100%;
  padding: 3px 0px;
  animation: fadeInOut 2s forwards;
  background: rgb(250, 100, 100);
  color: white;
}
```

## Button Loading State ClassNames

Initial State

```
base-class
```

When performing action

```
base-class, base-class-loading
```

* After action succeeded

```
base-class, base-class-success
```

After action failed
```
base-class, base-class-error
```

When performing another action
```
base-class, base-class-loading
```

repeat from *

## How to test this PR
- When you Post, Edit, Reply, Load more Comments or Load more replies, see that the appropriate loading classes are attached to the button.
- Also check that an error notification is shown when an error occurred
- The buttons should be in disabled state when they are `loading`.
- The comment body of a reply, edit, or post should only get erased AFTER the action was successful

### CODE REVIEW

File: locales/es.yml [hunk 1]
```diff
@@ -192,6 +191,7 @@ es:
     email_required: "Se requiere un correo"
     email_username_in_use: "Correo o nombre en uso."
     INVALID_ASSET_URL: "La URL del articulo no es valida"
+    unexpected_error: "Lo siento. Ha habido un error no previsto."
```
Comment (ALawliet) [inline locales/es.yml]: I noticed in `locales/en.yml`, `unexpected_error` was changed to `unexpected`.
https://github.com/coralproject/talk/blob/9de99a9ef45fe89e345af6951beb2f456fb7f39b/locales/en.yml#L209

I also noticed `msg = t('error.unexpected')` in `forEachError`.
https://github.com/coralproject/talk/blob/9de99a9ef45fe89e345af6951beb2f456fb7f39b/client/coral-framework/utils/index.js#L142.

In `locales/es.yml` you changed the translation, but did you mean to change the key to `unexpected` as well?

Comment (alinhle) [reply locales/es.yml]: 👍  Very careful observation! 💯

**Review by okbel — APPROVED** [review]

File: client/coral-embed-stream/src/components/LoadMore.js [hunk 2]
```diff
@@ -1,12 +1,10 @@
 import React, {PropTypes} from 'react';
```
Comment (okbel) [inline client/coral-embed-stream/src/components/LoadMore.js]: `import PropTypes from 'prop-types';`

Comment (alinhle) [reply client/coral-embed-stream/src/components/LoadMore.js]: Sure ^^

File: locales/es.yml [hunk 3]
```diff
@@ -193,6 +192,7 @@ es:
     email_required: "Se requiere un correo"
     email_username_in_use: "Correo o nombre en uso."
     INVALID_ASSET_URL: "La URL del articulo no es valida"
+    unexpected: "Lo siento. Ha habido un error no previsto."
```
Comment (okbel) [inline locales/es.yml:195]: You speak better than me haha 👍

Comment (alinhle) [reply locales/es.yml:195]: Nicht wahr :-D

**Pull request merged**